### PR TITLE
Feature: Fix for update block button visibilty on windows

### PR DESCRIPTION
**Pull request by bspeare**
# How to test

- `yarn workspace uids_base build`
- `blt ds --site=sandbox.uiowa.edu`
- Test on browserstack Chrome and verify that you can see the "update" block if editing a block in layout builder

### CODE REVIEW

**Comment by richardbporter** [comment]
I'm seeing a console error saying effectively `auto` is undefined. I think it should it be a string but then the `px` unit string should be refactored as well otherwise it gets written as `width: autopx;`.

**Comment by richardbporter** [comment]
Having some BrowserStack issues at the moment. I can't click into anything...

File: docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js [hunk 1]
```diff
@@ -11,7 +11,7 @@
                 let offCanvasWidth;
                 const offCanvasCookie = $.cookie('ui_off_canvas_width');
                 if (offCanvasCookie === undefined) {
-                  offCanvasWidth = 500;
+                  offCanvasWidth = auto;
```
Comment (richardbporter) [inline docroot/modules/custom/layout_builder_custom/js/layout_builder_custom.overrides.js]: Comment https://github.com/uiowa/uiowa/pull/4211#issuecomment-929602760 was in regards to this line.

**Comment by richardbporter** [comment]
Adding
 ```
.ui-dialog.ui-dialog-off-canvas {
  left: auto;
}
```

to layout_builder_custom.overrides.css was working for me locally even with offCanvasWidth set back to 500.

**Comment by richardbporter** [comment]
@bspeare can you try that out or some variation of it? Not 100% on that.

**Comment by bspeare** [comment]
@richardbporter thanks!  I will try it out now.

**Comment by bspeare** [comment]
@richardbporter do you want to test it again?  I think your suggestion worked.

**Comment by richardbporter** [comment]
Yea I was getting inconsistent results but I think that `!important` did the trick. Its not off the screen anymore: 
image

**Comment by richardbporter** [comment]
Ugh. Just tried this is Chrome on OSX and I can't drag the tray left to make it wider. 😢

**Comment by bspeare** [comment]
@richardbporter good testing!  Maybe there's another way we can set this.

**Review by richardbporter — APPROVED** [review]
9ede694 seems to work. Haven't noticed any issues with it.

**Comment by bspeare** [comment]
@richardbporter thanks for your help on this!

**Pull request merged**